packages/source-gdocs-paste/test/atjson-source-gdocs-translator-test.ts

```diff
@@ -0,0 +1,40 @@
+import GDocsSource from '@atjson/source-gdocs-paste';
+import * as fs from 'fs';
+import * as path from 'path';
+
+describe('@atjson/source-gdocs-paste', () => {
+  var atjson;
+
+  beforeAll(() => {
+    let fixturePath = path.join(__dirname, 'fixtures', 'complex.json');
+    let rawJSON = JSON.parse(fs.readFileSync(fixturePath));
+    let gdocs = new GDocsSource(rawJSON);
+    atjson = gdocs.toCommonSchema();
+  });
+
+  it('correctly converts -gdocs-ts_bd to bold', () => {
+    let bolds = atjson.annotations.filter(a => a.type === 'bold');
+    expect(bolds.length).toEqual(2);
+  });
+
+  it('correctly converts italic', () => {
+    let italics = atjson.annotations.filter(a => a.type === 'italic');
+    expect(italics.length).toEqual(2);
+  });
+
+  it('correctly converts headings', () => {
+    let headings = atjson.annotations.filter(a => a.type === 'heading');
+    expect(headings.length).toEqual(4);
+    expect(headings.map(h => h.attributes.level)).toEqual([1, 2, 100, 101]);
```

Comment:
100 / 101 🙃

Reply:
Yeah - we should change that (those are title/subtitle, not headings), but I'm not going to worry about it for the moment.